trace_viewer/extras/about_tracing/record_selection_dialog.html

```diff
@@ -323,8 +324,10 @@
         throw new Error('RecordSelectionDialog.currentlyChosenPreset:' +
             ' preset must be an array.');
       this.currentlyChosenPreset_ = preset;
+      this.isPresetSelected_ = false;
```

Comment:
Using only currentlyChosenPreset.length, causes problem in the border scenario where user has deselected all the categories.

Since the length is zero, following selections/deselections are completely ignored - not acted upon. The check for length in updateSetting_() cannot be removed since we want to modify the currentlyChosenPreset_ only for preset modes and not for manual mode.

Hence, the need for new boolean. Thoughts?